plugins/spu2-x/src/CMakeLists.txt

```diff
@@ -88,7 +88,6 @@ if(Windows)
     include_directories("Windows")
 else()
     LIST(APPEND spu2xSources
-        Linux/AboutBox.cpp
```

Comment:
It'd be a good idea to also remove Linux/AboutBox.cpp from the repo.

Reply:
I can't find it in my repo, I thought you removed that a while ago?